types/lib.bevy.d.ts

```diff
@@ -68,6 +68,7 @@ declare class World {
   resource<T>(type: BevyType<T>): T | null;
 
   query<Q extends QueryParameter[]>(...query: Q): QueryItem<Q>[];
+  get<Q extends QueryParameter[]>(entity: Entity, ...components: Q): MapQueryArgs<Q>;
```

Comment:
Maybe if there is only one argument, we just return one component instead of an array of one component?

That way calling `const myComponent = world.get(entity, MyComponent)` will work like Bevy, but you can still do the multiple-get thing if you want, which feels nicely similar to the pattern you use when querying.

Reply:
Good point, I'll separate it into `get` and `get_many`.

Reply:
OK, now I'm not so sure `get_many`, because I just ran into the case where of the the many components existed on the entity, but the others didn't.

What should the behavior be in that case? Eventually in queries we'll be able to specify optional components explicitly, so maybe it's less confusing and more bevy-aligned to have a single `get` method on `World`, and then support this pattern for a multiple-get on a single entity, just like Bevy:

```ts
const query = world.query(Transform, Velocity);
const [transform, velocity] = query.get(myEntity);
```